tools/pepsi/src/cargo.rs

```diff
@@ -27,6 +31,54 @@ pub enum Command {
 }
 
 pub async fn cargo(arguments: Arguments, repository: &Repository, command: Command) -> Result<()> {
+    if arguments.remote {
```

Comment:
Please consider factoring this part out to a separate function.

Reply:
Done